lib/semantics/semantics.cc

```diff
@@ -118,7 +119,8 @@ using StatementSemanticsPass1 = ExprChecker;
 using StatementSemanticsPass2 = SemanticsVisitor<AllocateChecker,
     ArithmeticIfStmtChecker, AssignmentChecker, CoarrayChecker,
     DeallocateChecker, DoChecker, IfStmtChecker, IoChecker, NullifyChecker,
-    OmpStructureChecker, PurityChecker, ReturnStmtChecker, StopChecker>;
+    OmpStructureChecker, PurityChecker, ReturnStmtChecker, StopChecker,
+    SelectStmtChecker>;
```

Comment:
This list of checkers was in alphabetical order.

Reply:
Oops, I didn't pay attention to the alphabetical order. Thanks for pointing it out.